### DISCUSSION TITLE
how to set prBodyDefinitions or  prBodyColumns to display Age in prBodyColumns

### DISCUSSION THREAD

**Discussion by zbyufei** [renovatebot/renovate#8124] [Request Help]
releaseTimestamp is already defined in datasource:maven,  but how to set prBodyDefinitions or  prBodyColumns to display `Age` in `prBodyColumns`

**Comment by rarkins** [comment]
First it would need to be calculated and then added to each update object. Then the field needs to be exposed to templates. After that you can use it.

**Reply by zbyufei** [reply]
OK, thinks.
I thought the age function was already implemented,
I follow https://github.com/renovatebot/renovate/pull/5960  to do it.
